VideoLocker/src/main/java/org/edx/mobile/http/cache/CacheManager.java

```diff
@@ -34,16 +32,13 @@ public CacheManager(Context context) {
         }
     }
 
-    public boolean has(String url) throws NoSuchAlgorithmException,
-            UnsupportedEncodingException {
+    public boolean has(String url) {
```

Comment:
It looks like url needs to be non-null, correct? (Or else an NPE will occur). If so, can you mark accordingly? In fact, even if it is nullable, can you mark accordingly?